components/shared/posts/NewContent.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useRef } from "react";
+import { Editor } from "@tinymce/tinymce-react";
+import { Controller } from "react-hook-form";
+interface NewContentProps {
+  register: any;
+  control: any;
+}
+const NewContent = ({ register, control }: NewContentProps) => {
+  const editorRef = useRef<any>(null);
+
+  return (
+    <section className="border-#4448691A border-t">
+      <Controller
+        control={control}
+        name="content"
+        render={({ field }) => (
+          <Editor
+            id="content"
+            onBlur={field.onBlur}
+            apiKey={process.env.NEXT_PUBLIC_TINYMCE_API_KEY}
```

Comment:
This is fine- but to be clear when we discussed this before: you could have kept this hardcoded. This doesn't protect your key and your key doesn't need to be protected. If you use "next_public_" it will end up in the frontend anyway.

:) 

Nothing needs to change! Just a heads up!